scapy/arch/bpf/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Guillaume Valadon <guillaume@valadon.net>
 
-"""
-Scapy *BSD native support
+r"""
+Scapy \*BSD native support
```

Comment:
What is this?

Reply:
This is supposed to be valid ReST. A `*` marks the beginning of bold text :/

Reply:
OK... Maybe "Scapy BSD native support" would work?